src/styles/variables.ts

```diff
@@ -81,7 +81,7 @@ export default {
     iconSizeXLarge: 28,
     iconSizeExtraLarge: 40,
     iconSizeSuperLarge: 60,
-    iconSizeUltraLarge: 120,
```

Comment:
Can you check and see where else this is used? Is it only used on the map page? If so, we're all good here.

Reply:
Yes we are all good here.